manifests/07-operator.yaml

```diff
@@ -54,6 +54,8 @@ spec:
               value: "cluster-image-registry-operator"
             - name: IMAGE
               value: docker.io/openshift/origin-docker-registry:latest
+            - name: IMAGE_PRUNER
+              value: openshift/cli:latest
```

Comment:
this value needs to be `quay.io/openshift/origin-cli:v4.0`

Reply:
to match what you put in image-references

Reply:
updated, thanks